script/export_for_foi/export.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+timestamp=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+
+echo 'Started site export'
+psql -d transition_development -f ./export_sites.sql > ./sites-$timestamp.csv
```

Comment:
Postgres authentication won't work like this in production - in dev this is relying on peer auth as the `vagrant` role. Making this into a rake task is probably the easiest way of creating all three CSVs at once. Or I can do it manually...